index.rst

```diff
@@ -14,14 +14,16 @@ infrastructure. It describes our tools and general operational procedures.
 
 Infrastructure Responsibility Matrix
 ------------------------------------
-The section aims to define ownership and point of contact for issues.
+The section aims to define ownership and point of contact for issues in
+the infrastructure team. Currently, there is 2 separates roles in order
+to let people focus on specific area.
 
-Gluster Sysadmin
-''''''''''''''''
+Sysadmin
+''''''''
 Michael Scherer
 
-Gluster CI Engineer
-'''''''''''''''''''
+CI Engineer
+'''''''''''
 Nigel Babu
```

Comment:
Add Deepshikha

Reply:
Then we also have to pluralize it somehow, I will defer that for next PR

Reply:
Well, then s/CI Engineer/CI Team/g